docs/configuration.md

```diff
@@ -209,10 +211,10 @@ Apart from these, the following properties are also available, and may be useful
   <td>(none)</td>
   <td>
     Set a special library path to use when launching the driver JVM.
-    
+
```

Comment:
Sean - to follow up on the white-space comment. The reason that there's white space changes is that there's a lot of trailing white space in these files which my IDE is stripping. I could disable white-space stripping but in this case, I think that this is a fix worth making since it shouldn't be there in the first place.